src/PresentationalComponents/SystemsTable/SystemsTable.js

```diff
@@ -42,16 +47,31 @@ const SystemsTable = ({ systemsFetchStatus, fetchSystems, systems, intl, filters
         2: 'last_seen'
     };
 
-    const onSort = (_event, index, direction) => {
+    const onSort = ({ index, direction }) => {
         const orderParam = `${direction === 'asc' ? '' : '-'}${sortIndices[index]}`;
         setSortBy({ index, direction });
         setFilters({ ...filters, sort: orderParam, offset: 0 });
     };
 
-    const onSetPage = (pageNumber) => {
-        const newOffset = pageNumber * filters.limit - filters.limit;
-        setFilters({ ...filters, offset: newOffset });
-    };
+    useEffect(() => {
+        (async () => {
+            const {
+                inventoryConnector,
+                mergeWithEntities,
+                INVENTORY_ACTION_TYPES
+            } = await insights.loadInventory({
+                react: React,
+                reactRouterDom,
+                pfReactTable
+            });
+            getRegistry().register({
+                ...mergeWithEntities(systemReducer(cols, INVENTORY_ACTION_TYPES))
+            });
+
+            const { InventoryTable } = inventoryConnector(store);
+            setInventory(() => InventoryTable);
+        })();
+    }, []);
```

Comment:
should this have the dependencies`[cols, store]`  ?

Reply:
Not really, this needs to happen only on initial render.

Reply:
ooo in that case would yah mind throwing ah  ` // eslint-disable-next-line react-hooks/exhaustive-deps` in the line above?

Reply:
Added, thanks for pointing it out!